<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <title>Chris Miller's Curriculum Vitae</title>

  <link rel="stylesheet" href="cv.css" type="text/css">
    <link href='http://fonts.googleapis.com/css?family=Cantarell' rel='stylesheet' type='text/css'>
</head>


<body>
  <div id="container">
    <div id="topbar">
      &nbsp;
    </div>


    <div id="header">
      <div id="toppic">
    	<img src="square.png" alt="Chris Miller" id="headshot">
      </div>
      <div id="title">
	<br />
    	<h1>Chris Miller, PhD</h1>
    	<h2>Analysis Manager - Cancer Genomics</h2>
    	<h2>The Genome Institute at Washington University</h2>
      </div>
      <div id="contact">
	<a href="mailto:chrisamiller@gmail.com">chrisamiller@gmail.com</a><br />
	<a href="http://www.chrisamiller.com">chrisamiller.com</a><br />
      </div>
    </div>


    <div class="twoColContainer">
    <hr>
    <div class="leftcol">
	<h3>About Me</h3>
      </div>
      <div class="rightcol">
	<p>I currently work in the Genome Institute at Washington University in St. Louis, 
	  where I lead a group that develops and applies computational tools 
	  to give us insight into the origins and progression of cancer.
	  </p><p>
	  Before moving to WashU, I completed a PhD in Computational Biology at Baylor 
	  College of Medicine. My thesis was about using bioinformatic techniques and large-scale 
	  data integration to help understand the aetiology of cancer.
	  </p><p> 

	</p>
	<p>My core research interests include:</p>
  	<ul>
  	  <li>Cancer - studying the evolutionary pressures driving tumorigenesis</li>
  	  <li>Integrative analysis - combining disparate sources of genomic data to place aberrations into a functional context</li>
  	  <li>Structural changes in the human genome and their contributions to disease</li>
  	  <li>Developing open-source software for integrating, mining, and visualizing genomic data</li>
  	  <li>Utilizing online technologies to drive science communication and collaboration</li>
	</ul>
      </div>
    </div>


    <div class="twoColContainer">
    <hr>
      <div class="leftcol">
	<h3>Education</h3>
      </div>
      <div class="rightcol">
	<ul>
	  <li>PhD, Structural and Computational Biology</br />
	  and Molecular Biophysics program<br />
	    Baylor College of Medicine<br />
	    Mentor: Dr. Aleksandar Milosavljevic<br />
	    Thesis: <a href="http://www.chrisamiller.com/science/Miller_Chapter1.pdf">Detection and Integrative Analysis of Genomic Aberrations in Cancer</a><br />
	    (graduated February 2011)
	  </li>
	  <li>BS degrees in Biology and Computer Science<br />
	    Truman State University<br />
	    (graduated May 2005)</li>
	</ul>
      </div>
    </div>

    <div class="twoColContainer">
    <hr>
      <div class="leftcol">
	<h3>Skills</h3>
      </div>
      <div class="rightcol">
	<ul>
	  <li>Designing genomic analysis pipelines using Perl, R, Bash, Ruby, and Linux</li>
	  <li>Synthesizing data from a wide variety of genomic techniques and technologies, including:
	    <ul>
	      <li id="sub">massively-parallel sequencing:<br />
		&nbsp;&nbsp;- paired-end sequencing for structural mapping<br />
		&nbsp;&nbsp;- RNAseq for transcriptome interrogation<br />
		&nbsp;&nbsp;- bisulfite sequencing for epigenomic assays<br />
		&nbsp;&nbsp;- copy-number inference from read depth<br />
		&nbsp;&nbsp;- SNP and indel detection<br />
	      </li>
	      <li id="sub">array comparative genomic hybridization (aCGH)</li>
	      <li id="sub">SNP, expression, and methylation arrays</li>
	      <li id="sub">functional and pathway databases</li>
	    </ul>
	  </li>
	  <li>Design of machine-learning tools for extracting patterns from genomic data</li>
	  <li>Utilizing high-performance linux clusters to expidite analyses</li>
	  <li>Data visualization and web design</li>
	  <li>Facilitating interdisciplinary collaborations</li>
	</ul>
      </div>
    </div>


    <div class="twoColContainer">
    <hr>
      <div class="leftcol">
	<h3>Publications</h3>
      </div>
      <div class="rightcol">
        <ul>
<li class="pub"><a href="http://www.cell.com/cancer-cell/fulltext/S1535-6108(14)00054-3">Functional Heterogeneity of Genetically Defined Subclones in Acute Myeloid Leukemia</a><br />
<span class="pubdetails">Klco JM, Spencer DH, <strong>Miller CA</strong>, Griffith M, Lamprecht TL, Oâ€™Laughlin M, Fronick C, Magrini V, Demeter RT, Fulton RS, <em>et al.</em>. Cancer Cell<br />
doi:10.1016/j.ccr.2014.01.031</span>
</li>
<li class="pub"><a href="http://www.nature.com/ncomms/2014/140122/ncomms4156/full/ncomms4156.html">Integrated analysis of germline and somatic variants in ovarian cancer</a><br />
<span class="pubdetails">Kanchi KL, Johnson KJ, Lu C, McLellan MD, Leiserson MD, Wendl MC, Zhang Q, Koboldt DC, Xie M, Kandoth C, <em>et al.</em>. Nature Communications<br />
doi:10.1038/ncomms4156</span>
</li>
<li class="pub"><a href="http://www.nejm.org/doi/full/10.1056/NEJMc1308782">Response to 'Genomic Landscapes and Clonality of De Novo AML'</a><br />
<span class="pubdetails"><strong>Miller CA</strong>, Wilson RK, Ley TJ. New England Journal Of Medicine<br />
doi:10.1056/NEJMc1308782</span>
</li>
<li class="pub"><a href="http://www.nature.com/nature/journal/v502/n7471/full/nature12634.html">Mutational landscape and significance across 12 major cancer types</a><br />
<span class="pubdetails">Kandoth C, McLellan MD, Vandin F, Ye K, Niu B, Lu C, Xie M, Zhang Q, McMichael JF, Wyczalkowski MA, <em>et al.</em>. Nature<br />
doi:10.1038/nature12634</span>
</li>
<li class="pub"><a href="http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2689.html">DGIdb: mining the druggable genome</a><br />
<span class="pubdetails">Griffith M, Griffith OL, Coffman AC, Weible JV, McMichael JF, Spies NC, Koval J, Das I, Callaway MB, Eldred JM, <em>et al.</em>. Nature Methods<br />
doi:10.1038/nmeth.2689</span>
</li>
<li class="pub"><a href="http://www.sciencedirect.com/science/article/pii/S2211124713004634">Endocrine-Therapy-Resistant< i> ESR1</i> Variants Revealed by Genomic Characterization of Breast-Cancer-Derived Xenografts</a><br />
<span class="pubdetails">Li S, Shen D, Shao J, Crowder R, Liu W, Prat A, He X, Liu S, Hoog J, Lu C, <em>et al.</em>. Cell Reports<br />
doi:10.1016/j.celrep.2013.08.022</span>
</li>
<li class="pub"><a href="http://genome.cshlp.org/content/23/3/431.long">Somatic neurofibromatosis type 1 (NF1) inactivation characterizes NF1-associated pilocytic astrocytoma</a><br />
<span class="pubdetails">Gutmann DH, McLellan MD, Hussain I, Wallis JW, Fulton LL, Fulton RS, Magrini V, Demeter R, Wylie T, Kandoth C, <em>et al.</em>. Genome Research<br />
doi:10.1101/gr.142604.112</span>
</li>
<li class="pub"><a href="http://www.nature.com/leu/journal/v27/n6/full/leu201358a.html">Clonal diversity of recurrently mutated genes in myelodysplastic syndromes</a><br />
<span class="pubdetails">Walter M, Shen D, Shao J, Ding L, White B, Kandoth C, <strong>Miller CA</strong>, Niu B, McLellan M, Dees N, <em>et al.</em>. Leukemia<br />
doi:10.1038/leu.2013.58</span>
</li>
<li class="pub"><a href="http://www.nejm.org/doi/full/10.1056/NEJMoa1301689">Genomic and Epigenomic Landscapes of Adult De Novo Acute Myeloid Leukemia</a><br />
<span class="pubdetails">The Cancer Genome Atlas Research Network. New England Journal Of Medicine<br />
doi:10.1056/NEJMoa1301689</span>
</li>
<li class="pub"><a href="http://www.cell.com/abstract/S0092-8674(12)00777-5">The Origin and Evolution of Mutations in Acute Myeloid Leukemia</a><br />
<span class="pubdetails">Welch JS, Ley TJ, Link DC, <strong>Miller CA</strong>, Larson DE, Koboldt DC, Wartman LD, Lamprecht TL, Liu F, Xia J, <em>et al.</em>. Cell<br />
doi:10.1016/j.cell.2012.06.023</span>
</li>
<li class="pub"><a href="http://www.sciencedirect.com/science/article/pii/S1934590912001191">Background mutations in parental cells account for most of the genetic heterogeneity of induced pluripotent stem cells</a><br />
<span class="pubdetails">Young MA, Larson DE, Sun C, George DR, Ding L, <strong>Miller CA</strong>, Lin L, Pawlik KM, Chen K, Fan X, <em>et al.</em>. Cell Stem Cell<br />
doi:10.1016/j.stem.2012.03.002</span>
</li>
<li class="pub"><a href="http://genome.cshlp.org/content/early/2012/02/02/gr.129684.111.abstract">VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing</a><br />
<span class="pubdetails">Koboldt DC, Zhang Q, Larson DE, Shen D, McLellan MD, Lin L, <strong>Miller CA</strong>, Mardis ER, Ding L, Wilson RK. Genome Research<br />
doi:10.1101/gr.129684.111</span>
</li>
<li class="pub"><a href="http://www.nature.com/nature/journal/v486/n7403/full/nature11143.html">Whole-genome analysis informs breast cancer response to aromatase inhibition</a><br />
<span class="pubdetails">Ellis MJ, Ding L, Shen D, Luo J, Suman VJ, Wallis JW, Van TBA, Hoog J, Goiffon RJ, Goldstein TC, <em>et al.</em>. Nature<br />
doi:10.1038/nature11143</span>
</li>
<li class="pub"><a href="http://www.nature.com/nature/journal/v481/n7382/full/nature10738.html">Clonal evolution in relapsed acute myeloid leukaemia revealed by whole-genome sequencing</a><br />
<span class="pubdetails">Ding L, Ley TJ, Larson DE, <strong>Miller CA</strong>, Koboldt DC, Welch JS, Ritchey JK, Young MA, Lamprecht T, McLellan MD, <em>et al.</em>. Nature<br />
doi:10.1038/nature10738</span>
</li>
<li class="pub"><a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002216">BioStar: an online question & answer resource for the bioinformatics community</a><br />
<span class="pubdetails">Parnell LD, Lindenbaum P, Shameer K, Dall'Olio GM, Swan DC, Jensen LJ, Cockell SJ, Pedersen BS, Mangan ME, <strong>Miller CA</strong>, <em>et al.</em>. PLoS Computational Biology<br />
doi:10.1371/journal.pcbi.1002216</span>
</li>
<li class="pub"><a href="http://www.sciencedirect.com/science/article/pii/S2210776211001943">Long-range massively parallel mate pair sequencing detects distinct mutations and similar patterns of structural mutability in two breast cancer cell lines</a><br />
<span class="pubdetails">Hampton OA, <strong>Miller CA</strong>, Koriabine M, Li J, Den HP, Carbone L, Nefedov M, Ten HBF, Lee AV, De JPJ, <em>et al.</em>. Cancer Genetics<br />
doi:10.1016/j.cancergen.2011.07.009</span>
</li>
<li class="pub"><a href="http://www.biomedcentral.com/1755-8794/4/34">Discovering functional modules by identifying recurrent and mutually exclusive mutational patterns in tumors</a><br />
<span class="pubdetails"><strong>Miller CA</strong>, Settle SH, Sulman EP, Aldape KD, Milosavljevic A. BMC Medical Genomics<br />
doi:10.1186/1755-8794-4-34</span>
</li>
<li class="pub"><a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0016327">ReadDepth: a parallel R package for detecting copy number alterations from short sequencing reads</a><br />
<span class="pubdetails"><strong>Miller CA</strong>, Hampton O, Coarfa C, Milosavljevic A. PLoS One<br />
doi:10.1371/journal.pone.0016327</span>
</li>
<li class="pub"><a href="http://www.biomedcentral.com/1471-2105/11/572">Pash 3.0: A versatile software package for read mapping and integrative analysis of genomic and epigenomic variation using massively parallel DNA sequencing</a><br />
<span class="pubdetails">Coarfa C, Yu F, <strong>Miller CA</strong>, Chen Z, Harris RA, Milosavljevic A. BMC Bioinformatics<br />
doi:10.1186/1471-2105-11-572</span>
</li>
<li class="pub"><a href="http://genome.cshlp.org/content/19/2/167.long">A sequence-level map of chromosomal breakpoints in the MCF-7 breast cancer cell line yields insights into the evolution of a cancer genome</a><br />
<span class="pubdetails">Hampton OA, Den HP, <strong>Miller CA</strong>, Delgado DA, Li J, Coarfa C, Harris RA, Richards S, Scherer SE, Muzny DM, <em>et al.</em>. Genome Research<br />
doi:10.1101/gr.080259.108</span>
</li>
<li class="pub"><a href="http://www.nature.com/nature/journal/v455/n7216/full/nature07385.html">Comprehensive genomic characterization defines human glioblastoma genes and core pathways</a><br />
<span class="pubdetails">McLendon R, Friedman A, Bigner D, Van MEG, Brat DJ, Mastrogianakis GM, Olson JJ, Mikkelsen T, Lehman N, Aldape K, <em>et al.</em>. Nature<br />
doi:10.1038/nature07385</span>
</li>
        </ul>
      </div>
    </div>


    <div class="twoColContainer">
    <hr>
      <div class="leftcol">
	<h3>Posters and Presentations</h3>
      </div>
      <div class="rightcol">
	<ul>
	  <li>Multi-dimensional clustering of mutations from heterogeneous tumors offers insight into clonal architecture, tumor evolution, and response to therapy<br />
	    Presentation: The Cancer Genome Atlas Steering Committee Meeting (2013)
	  </li>

	  <li>Assessing Tumor Heterogeneity and Tracking Clonal Evolution Using Whole Genome or Exome Sequencing<br />
	    Presentation: The Cancer Genome Atlas Scientific Symposium (2012) <a href="http://cancergenome.nih.gov/newsevents/multimedialibrary/videos/AssessingMiller2012">link</a></li>

	  <li>Identification of driver mutations in non-coding functional regions of cancer genomes<br />
	  Poster: Genome Informatics (2012)</li>

	  <li>Discovering functional modules relevant for cancer progression by identifying patterns of recurrent and mutually exclusive mutations in tumor samples
	    <br />Poster: CSHL Biology of Genomes (2010)</li>

	  <li>Integrative analysis of multi-dimensional genomic data identifies key relationships between cancer pathways<br />
	    Poster: SCBMB Retreat (2008)</li>

	  <li>Pash: Efficient Genome-Scale Sequence Anchoring by Positional Hashing
	    <br />Poster: Rice University Computational Biology Conference (2006)</li>

	  <li> An Image Analytic Approach to Characterizing Vasculogenic Networks
	    <br />Presentation: Truman State University Student Research Symposium (2005)</li>	  	</ul>
      </div>
    </div>


    <div class="twoColContainer">
    <hr>
      <div class="leftcol">
      <h3>Teaching</h3>
      </div>
      <div class="rightcol">
      <ul>
	<li>Computer Aided Discovery Methods (2007 and 2008)<br />
	  Contributed to the development of the course and taught computational labs on:
	  <ul>
	    <li id="sub">Studies of genome variation</li>
	    <li id="sub">Transcriptional regulatory pathways</li>
	  </ul>
	</li>
	<li>Computational Mathematics for Biomedical Scientists (2008)
	  <ul>
	    <li id="sub">Teaching Assistant</li>
	  </ul>
	</li>
	<li>Advanced Topics: Quantitative Biology (2005)
	  <ul>
	    <li id="sub">Teaching Assistant</li>
	  </ul>
	</li>

      </ul>
      </div>
    </div>


    <div class="twoColContainer">
    <hr>
      <div class="leftcol">
	<h3>Past Research</h3>
      </div>
      <div class="rightcol">
	   <ul>
	     <li>Classification and analysis of Alu SINEs in Human and Chimpanzee Chromosomes 3 and 12<br />
	       Mentor: Dr. Aleksandar Milosavljevic (BCM)<br />
	       Rotation project - Fall 2005</li>

	     <li>Development of an open-source image analysis toolset for describing the structure and dynamics of biological tissue.<br />
	       Mentors: Dr Jason Miller (Truman State University) and Dr Robert Baer (Kirksville College of Osteopathic Medicine)<br />
	       Summer 2004 - Spring 2005</li>

	   </ul>
    </div>


    <div class="twoColContainer">
    <hr>
      <div class="leftcol">
	<h3>Other</h3>
      </div>
      <div class="rightcol">
	<ul>
	  <li>I write about the intersection of science, technology, and culture at <a href="http://www.chrisamiller.com/science/">http://www.chrisamiller.com/science/</a></li>
	  <li>BCM Graduate Student Council<br />
	    &nbsp;&nbsp;&nbsp;- Vice-president 2008<br />
	    &nbsp;&nbsp;&nbsp;- SCBMB representative 2007
	  </li>
	  <li>Member of Phi Sigma Pi National Honor Fraternity<br />&nbsp;&nbsp;&nbsp;- Chapter Service Key recipient (2004)</li>
	  <li>Eagle Scout</li>
	</ul>
	</div>
    </div>
    </div>



<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script><script src="./index2_files/ga.js" type="text/javascript"></script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-7136507-1");
pageTracker._trackPageview();
} catch(err) {}</script>




</div></body></html>
